packages/shared/src/components/Notion/Blocks/Image.client.tsx

```diff
@@ -7,27 +7,20 @@ function Image({ ...props }) {
     props
 
   const isPriority = props?.priority ? props?.priority : order < 2 ? true : false
```

Comment:
🤔  Think this is checking if the `props?.variety` is true not necessarily that it exists. But this happens to works regardless.